build.rs

```diff
@@ -0,0 +1,85 @@
+use std::env;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+use maplit::hashmap;
+use memofs::VfsSnapshot;
+
+fn snapshot_from_fs_path(path: &PathBuf) -> Result<VfsSnapshot, PathBuf> {
+    if path.is_dir() {
+        let entries: Result<Vec<fs::DirEntry>, PathBuf> = fs::read_dir(path)
+            .map_err(|_| path.to_owned())?
+            .map(|entry| entry.map_err(|_| path.to_owned()))
+            .into_iter()
+            .collect();
```

Comment:
We should be able to keep these items as an iterator instead of collecting them. If you want better error messages with paths, [fs-err](https://docs.rs/fs-err/2.3.0/fs_err/) is a crate we recently added a dependency on.